### ISSUE TITLE
Restore purchases broken

### ISSUE THREAD

**Issue by 65fastback2plus2** [hymns-for-worship/issues#19]
The update prior to this one would never remember the restore purchases so you had to do it every time you used the app.

This latest update quit restoring purchases altogether.

Android, fully updated pixel 7

**Comment by rodydavis** [comment]
Thanks for letting me know! I am testing on android. There is a bug on the google play api and I'm looking into it. 


I'll report back here with what I find out!

**Comment by ElijahSink** [comment]
Can confirm—the app isn't usable for me on a Galaxy S21 Ultra. In attempting to follow the purchase-again-no-charge instructions, I actually was charged again. Despite being charged again, I still can't access any of the paid material.

**Comment by rodydavis** [comment]
This must be an issue with the google play api. I will file an issue with them. In the meantime refunds for duplicated purchases can be requested through google play. 

Thanks for confirming the issue!

**Comment by NeoShader** [comment]
Cannot open revised, supplement, or echoice. All say purchase required. Clicking purchase says I have them. Clicking restore does nothing. Reinstalled app, no dice.

OnePlus 9 PRO 5G
Android 13

**Comment by rodydavis** [comment]
Working on this issue, thanks for the confirmation!

**Comment by rodydavis** [comment]
Great news! Bug is found, and will be pushing a new update out shortly!

**Comment by rodydavis** [comment]
As per this documentation, after the new version is deployed (will be today) it can take up to 36 hours for the new credentials of the backend to be active.

https://www.revenuecat.com/docs/creating-play-service-credentials

Stay tuned and all should be good!

**Comment by rodydavis** [comment]
TLDR: Google Play API changed to where Firebase needed to be connected.

**Comment by rodydavis** [comment]
Fixed on android version: `2.1.17+527`

Ios in review for similar changes.
